packages/hawtio/src/plugins/shared/jolokia-service.ts

```diff
@@ -309,8 +309,7 @@ class JolokiaService implements IJolokiaService {
   }
 
   private ajaxError(resolve?: AjaxErrorResolver): JQueryAjaxError {
-    const errorThreshold = 2
```

Comment:
I still think the threshold is needed. It's used to ignore first 2 failures for some unknown reasons to make the connection robust. If we removed this, I wonder if it would then show error toast more frequently.

Reply:
ah sure - I'll display once, keep the status in the header (which was my main change), but display the toast after the threshold.

Reply:
@tadayosi how about now?